packages/strapi-plugin-content-manager/admin/src/components/InputUID/index.js

```diff
@@ -59,9 +58,10 @@ const InputUID = ({
   const wrapperRef = useRef(null);
   const generateUid = useRef();
   const initialValue = initialData[name];
-  const isCreation = isEmpty(initialData);
+  const createdAtName = get(layout, ['schema', 'options', 'timestamps'], ['created_at'])[0];
```

Comment:
You can directly use the following path `get(layout, ['schema', 'options', 'timestamps', 0])` the you can still provide a default value, however it differs when using the `mongoose` connector. Anyway, I think the back-end will always send us the timestamps.